core/tests/performance_tests/base.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common utilities for performance test classes"""
+
+import unittest
+
+from core.tests.performance_framework import selenium_driver
+from core.tests.performance_framework import process_data
+
+# pylint: disable=invalid-name
+selenium_data_fetcher = selenium_driver.SeleniumPerformanceDataFetcher
```

Comment:
No reason in particular, saw it being done this way in some other file. Also, the class name is pretty long!